cmd/legacy/run.go

```diff
@@ -123,6 +123,7 @@ func Run(cmd *cobra.Command, v *viper.Viper) {
 func SetupLogging(v *viper.Viper) {
 	logfileParams, err := logfile.LoadParams(v)
 	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to load log params: %s", err)
```

Comment:
Doesn't `log.*` package write to console (output) when not already setup?

Reply:
Oh maybe, but I like to be explicit. Also I wasn't sure if it would write to stdout or stderr, and we want errors always written to stderr.